lib/loader/index.d.ts

```diff
@@ -7,9 +7,9 @@ interface ImportsObject extends Record<string, any> {
   env?: {
     memory?: WebAssembly.Memory,
     table?: WebAssembly.Table,
-    abort?: (msg: number, file: number, line: number, column: number) => void,
-    trace?: (msg: number, numArgs?: number, ...args: any[]) => void
-  }
+    abort?(msg: number, file: number, line: number, column: number): void,
+    trace?(msg: number, numArgs?: number, ...args: number[]): void
+  };
```

Comment:
Semicolon seems wrong here

Reply:
Strange, but ts-lint suggest me write semicolon here.
Also semicolons present [here](https://github.com/AssemblyScript/assemblyscript/blob/bb31c0ee66187d5c1732a24d2cc11ef3fba8453d/lib/loader/index.d.ts#L33) for example. I also think this correct

Reply:
Because 
```ts
env?: {
  memory?: WebAssembly.Memory
  ...
};
```
is actually type and could be simplify to:
```ts
{
  env?: object;
}
```
So we need semicolon here

Reply:
Oh, right :)